utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/extractor/JsonTypedMap.scala

```diff
@@ -18,4 +19,12 @@ final case class JsonTypedMap(jsonObject: JsonObject, definition: SwaggerObject,
             JsonToNuStruct(jsonObject(key).getOrElse(Json.Null), swaggerType, if (path.isEmpty) key else s"$path.$key")
           case None => JsonToObjectError(jsonObject.asJson, definition, path)
         }
-    )
+    ) {
+
+  def materialize: TypedMap =
```

Comment:
This will be faster if you pass the new map a correct size hint in its constructor. Any additional `asScala` / `asMap` will only make this slower.